gamemode/index.js

```diff
@@ -279,7 +279,7 @@ const weatherSets = [
   "mm_210_gallery_cp_050",
 ];
 
-sdk.on("resourceLoaded", (name) => {
+sdk.on("gamemodeLoaded", () => {
```

Comment:
I don't really understand the point of this event because the file is executed immediately. (eg. console log line 1)

Reply:
It's useful when you want to perform a soft reload of the game mode. As opposed to re-initialising the VM.